server/src/models/staff/staff.dto.ts

```diff
@@ -47,14 +47,17 @@ export class UpdateStaffDto {
 
   @IsEmail()
   @IsNotEmpty()
+  @IsOptional()
```

Comment:
For updating whatever field you pass that is present, we update it, otherwise we keep the old value.

That way for example if I just want to update the first name, I don't need to pass in the rest of the information like lsat name, gender, etc

Reply:
Hmm I'm not sure about this, just because we want to keep the HTTP verb consistent with the behavior of the endpoint. This looks a lot like a `PATCH` and not a `PUT` now, I recommend that you just have a new endpoint for patching the user's password